describe/describe.go

```diff
@@ -0,0 +1,94 @@
+package describe
```

Comment:
not sure, but maybe I can just copy these all into cmd/describe...

Reply:
I did that because I knew we'd import the human readable format into other projects. I knew ach-web-viewer was something I wanted to created.